package-lock.json

```diff
@@ -4,6 +4,7 @@
     "requires": true,
     "packages": {
         "": {
+            "name": "entitlementcard",
```

Comment:
This gets removed if I run `npm i` :thinking:

Reply:
hm okay this was added by the audit

Reply:
okay i reverted this change